website/docs/docs/build/packages.md

```diff
@@ -165,27 +165,45 @@ dbt Cloud supports private packages from [supported](#prerequisites) Git repos l
 
 #### Prerequisites
 
-To use native private packages, you must have one of the following Git providers configured in the **Integrations** section of your **Account settings**:
-- [GitHub](/docs/cloud/git/connect-github)
-- [Azure DevOps](/docs/cloud/git/connect-azure-devops)
-- Support for GitLab is coming soon.
-
+- To use native private packages, you must have one of the following Git providers configured in the **Integrations** section of your **Account settings**:
+  - [GitHub](/docs/cloud/git/connect-github)
+  - [Azure DevOps](/docs/cloud/git/connect-azure-devops)
+    - For Azure DevOps, use the `org/repo` path (not the `org_name/project_name/repo_name` path) with the project tier inherited from the integrated source repository.
+    - Private packages only work within a single Azure DevOps project. If your repositories are in different projects within the same organization, you can't reference them in the `private` key at this time.
+  - Support for GitLab is coming soon.
 
 #### Configuration
 
-Use the `private` key in your `packages.yml` or `dependencies.yml` to clone package repos using your existing dbt Cloud Git integration without having to provision an access token or create a dbt Cloud environment variable:
+Use the `private` key in your `packages.yml` or `dependencies.yml` to clone package repos using your existing dbt Cloud Git integration without having to provision an access token or create a dbt Cloud environment variable. 
+
+The private key supports a two-layered-path: `org/repo`. dbt inherits the project path from your source repository.
```

Comment:
I would just leave this out - the project is kind of a concept specific to ADO, so it's not exactly correct to say that that's the behavior for other providers.